pyseir/cli.py

```diff
@@ -146,30 +160,47 @@ def _build_all_for_states(
         root.info("Only executing for states. returning.")
         return
 
-    all_county_fips = build_counties_to_run_per_state(states, fips=fips)
+    # Get the Lists of Fips to Calculate
+    pyseir_target_fips = build_counties_to_run_per_state(states, fips=fips)
+
+    infer_rt_target_fips = [fips] if fips else combined_datasets.load_us_latest_dataset().all_fips
```

Comment:
why go back to combined_datasets to get all fips instead of using pyseir_target_fips + states_regions ?

Reply:
The inclusion criteria is different for pyseir than for the infection rate calculation. Essentially, the infection rate calculation can tolerate lower absolute numbers but is more sensitive to needing recent data. In the past, the infection whitelist was a subset of the pyseir SEIR whitelist. That was mostly historical convenience, and I am more explicitly separating that going forward.